### ISSUE TITLE
Error parsing Layer "Error in $[0].Attributes: expected Text, encountered Null"

### ISSUE THREAD

**Issue by FintanH** [brendanhay/amazonka#396]
Using version: 1.4.5
When calling DescribeLayers I was receiving `_serializeMessage = "Error in $[0].Attributes: expected Text, encountered Null"`. 

The raw JSON looked like:
```
  "Attributes": {
    "BundlerVersion": null,
    "EcsClusterArn": null,
    "EnableHaproxyStats": null,
    "GangliaPassword": null,
    ...
```

I tracked the cause of this down to: https://github.com/brendanhay/amazonka/blob/develop/amazonka-opsworks/gen/Network/AWS/OpsWorks/Types/Product.hs#L1838

I have a fix that I can confirm works within my application: https://github.com/FintanH/amazonka/commit/36140aaaa8060cc7bc2540210753e1eb158af74d

@brendanhay Could you advise on how to follow up with merging the fix?

**Comment by dfithian** [comment]
Just ran into this, can we bring in the change @brendanhay ?

**Comment by brendanhay** [comment]
The generator overrides contain this fix already and it should be part of the `1.6.0` release, feel free to reopen if you still have problems.
